journal/updated/01-journal.adoc

```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
-prevailed with him to let them go,
-and he took our words for our appearance at the next court session.
-
-Through the goodness of God, we all returned together to Shrewsbury, to our prison,
+to see whether we could be granted the permission
+to go home till the next court session; but it was not then granted,
+although he told me that if I pleased I might go home till then.
+I told him he might as freely let them go as me, for most of them lived in the county,
+and I lived out of the county; but we could get nothing more from him at that time.
+I was uneasy in myself to leave him alone, seeing he had a regard for me,
+and so continued to press for my friends`' further liberty.
+
+A little time afterwards, by the argument that it was harvest time,
+and hard for the mens`' wives, or some of their families,
+to obtain their weekly necessities, I, with my friend John Millington,
+prevailed with him to let us go,
+him taking our word that we would appear at the next court session.
+
+Through the goodness of God, we all returned together to our prison at Shrewsbury
 before the court session, and found a great alteration in the jailer;
 he was very low and mournful.
 He had lost a prisoner, a malefactor, and was to be tried for his life for his escape.
-He was very loving and kind to us, and let Friends go themselves to Bridgenorth,
-about fourteen miles, where the court sessions were then held;
-and he desired me to stay with him in his affliction, and not be much from him.
+He was very loving and kind to us, and let Friends go alone to Bridgenorth,
+a distance of about fourteen miles, where the court sessions were then held;
+and he desired me to stay with him in his affliction, and not be much absent from him.
 He said his life was at stake, and if God and the judge would show him any mercy,
-it was upon our Friends`' account, and not for any deserts that were in himself,
-for he confessed he had been too severe to us; but notwithstanding, said he,
-you are merciful men, and can forgive wrongs and injuries.
+it was upon our Friends`' account, and not for any goodness that was in himself,
+for he confessed he had been too severe to us; "`but notwithstanding,`" said he,
+"`you are merciful men, and can forgive wrongs and injuries.`"
 
 When we came to Bridgenorth,
-we were put in a large spacious room in the house of correction,
-to be there in the day time, that we might be all together, and ready when called for;
-but we had liberty of going in and out for lodging, and what necessaries we needed;
-no keeper being over us, but what we set ourselves to look to the door,
-and that too many Friends might not be out at once,
-and those were not to stay out too long.
-We saw it was convenient, that Friends should go out by two and two, to walk the streets;
+we were put in a large, spacious room in the house of correction.
+We arrived there in the day time, so that we might all be together and ready when called for;
```

Comment:
The idea is: we were put in a large, spacious room in the house of correction, **in order to be there during the day time,** that we we might all be together and ready when called for...